manifest.json

```diff
@@ -1018,6 +1018,16 @@
                 }
             },
             "versions": {
+                "1.0.3": {
```

Comment:
ExpressionEvaluator seems to be merged into the assembly, so would need to review all of it too with this PR. Which at least I won't have the time/motivation to do for a while.